src/main/resources/messages.properties

```diff
@@ -1,61 +1,69 @@
 templates.vote-type.overseas=
```

Comment:
Does this template need to be filled in? Do you know where it is used?

Reply:
I think this is used to get the `sourceType` for the template but in overseas we don't have a `sourceType` in our templates as they are all overseas specific